xikolo-common/Data/Model/Channel.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Channel.swift
+//  xikolo-ios
+//
+//  Created by Bjarne Sievers on 27.10.16.
+//  Copyright © 2016 HPI. All rights reserved.
+//
+
+import Foundation
+import CoreData
+import Spine
+import UIKit
+
+class Channel: BaseModel {
+
```

Comment:
Since you're removing it, please delete the extra blank line (only leave 1).
If you had implemented it, always use `UIColor`, and then, in the places that do expect a `CGColor`, just use `color.CGColor`.